easybuild/easyblocks/s/scipy.py

```diff
@@ -211,4 +223,9 @@ def sanity_check_step(self, *args, **kwargs):
             'dirs': [det_pylibdir()],
         }
 
-        return PythonPackage.sanity_check_step(self, custom_paths=custom_paths)
+        # make sure that scipy is included in output of 'pip list',
+        # so that 'pip check' passes if scipy is a required dependency for another Python package;
+        # use case-insensitive match, since name is sometimes reported as 'SciPy'
+        custom_commands = [r"pip list | grep -i '^scipy[ \t]*%s[ \t]*$'" % self.version]
```

Comment:
you can keep the `\s` if you use extended regex
worth noting that we're not matching the `.` in the version string literally
```suggestion
        custom_commands = [r"pip list | grep -iE '^scipy\s+%s\s*$'" % self.version.replace('.', '\.')]
```